### DISCUSSION TITLE
Combine pipe immediately opening when CP harvest job started

### DISCUSSION THREAD

**Discussion by Fury1SOG** [Courseplay/Courseplay_FS22#2689] [Q&A]
Hello,
I am experiencing a strange problem where certain harvesters (not all) immediately open their pipe when a CoursePlay harvest job is started. This causes the combine to hit the trees next to the field. 

Once this happens, changes to CP or AD settings do not help. I discovered that deleting the <FS22_Courseplay> settings for the combine in vehicles.xml fixes it, but it will occur again.

Is there any setting that I am missing? 

Thank you very much for your advice.

**Comment by Tensuko** [comment]
It comes from AD, not from CP.
Example of combine where it happens?
Also when you have a trailer next to the field and the combine passes this, ofc it will open the pipe, thats a base game function!

**Reply by Fury1SOG** [reply]
Thank you very much for your reply.
The issue occurs with the Colossus standard and root harvesters  (https://www.farming-simulator.com/mod.php?lang=en&country=us&mod_id=253608&title=fs2022).
It seems to occur often when using multiple tools.

>Also when you have a trailer next to the field and the combine passes this, ofc it will open the pipe, thats a base game function!

Yes, I have seen this behavior too, but the issue I am experiencing is that as soon as the worker is assigned, the pipe opens. If I dismiss the worker and manually close the pipe, the pipe automatically opens again once a new worker is assigned.

**Reply by rohne83** [reply]
Are you using AD as unloaders? Then the combine will always open the pipe as soon as it reaches the set filllevel for combine chasing - that's AD feature with nasty side effects.

**Comment by Fury1SOG** [comment]
> Are you using AD as unloaders? Then the combine will always open the pipe as soon as it reaches the set filllevel for combine chasing - that's AD feature with nasty side effects.

Thank you for your reply. I was using CP's unloader. However, the AD HUD was active (AD start button not active).

In order for AD to immediately open the pipe after harvest start, would the fill level need to be set to 0%?
